support/cas-server-support-oauth/src/main/java/org/apereo/cas/config/CasOAuth20Configuration.java

```diff
@@ -356,6 +364,18 @@ public OAuth20CallbackAuthorizeViewResolver callbackAuthorizeViewResolver() {
             defaultPrincipalResolver.getObject());
     }
 
+    @ConditionalOnMissingBean(name = "oAuthRefreshTokenAuthenticator")
+    @Bean
+    @RefreshScope
+    public Authenticator<UsernamePasswordCredentials> oAuthRefreshTokenAuthenticator() {
```

Comment:
`oauthRefreshTokenAuthenticator`, and same for the annotation.

Reply:
Are you sure ? All the other authenticators have this syntax:
- oAuthProofKeyCodeExchangeAuthenticator
- oAuthClientAuthenticator
- oAuthAccessTokenAuthenticator
- etc...